.travis.yml

```diff
@@ -12,7 +12,8 @@ before_install:
   - ./run.sh bootstrap
 
 install:
-  - ./run.sh install_aptget r-cran-rcpp r-cran-runit r-cran-knitr r-cran-rmarkdown
+  - ./run.sh install_r Rcpp
```

Comment:
I _think_ can also install this via apt-get as the Rutter PPAs (which `run.sh` looks at) has current versions.

At least I never install from source.